### PR TITLE
Create new workflow to hide artifact links

### PR DESCRIPTION
**Pull request by McGiverGim**
This fixes the hide artifact links comments, that is givin an error.

The permissions for any workflow thrown by a PR are read only ALWAYS. 

This is a workaround, executing the action as a principal workflow outside of the PR.

### CODE REVIEW

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — DISMISSED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by McGiverGim** [comment]
Modified to remove the `opened` type of event. I think it has no sense to "hide" comments when the PR is open because it does not have any link to hide. Now only does it if the branch is modified (`synchronize`) or the PR is reopened (`reopened`) and I'm not too sure if the second one is needed.

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by ctzsnooze — APPROVED** [review]

**Pull request merged**
